Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3454.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 3454, "Picker accessibility text is wrong", PlatformAffected.macOS)]
+	public class Issue3454 : TestContentPage
```

Comment:
Can you run a *format document* over this issue? So all the brackets can be on the same line and the spaces between name and parenthesis can get removed?